data/api_v1_to_v2_test.go

```diff
@@ -127,7 +158,8 @@ func TestAPI_GetDataV2(t *testing.T) {
 	resetOPAMockRouteV1(true, "/v1/dataV2", userID)
 	handlerLogFunc := tidewhisperer.middlewareV1(tidewhisperer.getDataV2, true, "userID")
 
-	request, _ := http.NewRequest("GET", "/v1/dataV2/"+userID, nil)
+	// testing with cbg and basal buckets
+	request, _ := http.NewRequest("GET", "/v1/dataV2/"+userID+"?basalBucket=true&cbgBucket=true", nil)
```

Comment:
Not possible to omit param to get all data ? Like it's done today (to ensure retro compatibility)

Reply:
not sure it would make sense
Or we need to agree on default beahvior for this call